### DISCUSSION TITLE
Re-request review cancels standing requests

### DISCUSSION THREAD

**Discussion by FGrose** [community/community#40763] [Pull Requests]
### Select Topic Area

Bug

### Body

When a Re-request review button next to a reviewers name is pushed, pending or standing review requests by others are cancelled.  This is  unexpected.
https://github.com/dracutdevs/dracut/pull/2092 shows a history of this.
Instructions say I can convert the PR to a draft, and then back to auto request owner reviews. I tried that. --
The effect of converting to draft was to remove the existing tags and set the 'draft' tag, and also to auto request owner reviews. But then, after marking the PR for review, the auto requested owner reviews were dropped again, only leaving the supplementary reviewers.

**Comment by ossanna16** [comment]
Hi there @FGrose and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
